core/src/components/banner/banner.tsx

```diff
@@ -99,7 +99,7 @@ export class TdsBanner {
   render() {
     const usesHeaderSlot = hasSlot('subheader', this.host);
     const usesSubheaderSlot = hasSlot('subheader', this.host);
-    const usesBottomSlot = hasSlot('bottom', this.host);
```

Comment:
No link visible in the Banner, looks like it still uses "bottom" slot in the stories file?
![image](https://github.com/scania-digital-design-system/tegel/assets/48509966/46c5f6c7-f361-4cce-892e-54819e078295)